### DISCUSSION TITLE
RAM and CPU utilization stays the exact same as defaults after adjusting them post install

### DISCUSSION THREAD

**Discussion by GG-GhostGaming** [tteck/Proxmox#1896] [General]
When I change the allocated amount of RAM/CPU that is dedicated to the jellyfin LXC, it does not change within jellyfin or any other applications run inside of the same LXC. They all are maxed out at two gigs and two CPU cores. No matter what I do, I cannot seem to get it to go any higher even though I'm running both jellyfin and qBitTorrent in the same LXC. It never goes above two gigs (1.67) of RAM and two CPU cores (18% utilization on the CPU). (I have eight cores dedicated to the LXC and 10 gigs of RAM.)

After using the script successfully and gaining access to the web UI stop/shut down the jellyfin server, adjust CPU and RAM. Then check the summary on proxmox. If replicated properly you will notice it does not go above. 2. Gigs on RAM and has extremely low utilization on the CPU If you allocate 8 cores.

The OS is Ubuntu 20.04 or Ubuntu 22.04
And I have restarted/stopped and started the LXC multiple times.

![Screenshot 2023-10-08 004315](https://github.com/tteck/Proxmox/assets/55955972/7967ee64-cf8d-4dfe-8e3c-5fd28aa49740)

**Comment by tteck** [comment]
> They all are maxed out at two gigs and two CPU cores.

How are you checking this in the applications?

**Reply by GG-GhostGaming** [reply]
Proxmox never reports anything higher than 2 gigs being used and 18% CPU utilization, another way that I tell that it isn't just proxmox not reporting things properly is by running both qBitTorrent and streaming 1080p non-transcoded content on jellyfish at the same time. It becomes very unresponsive on both applications and I have to turn off one or the other in order to free up resources which should not be constrained.
